account/account_test.go

```diff
@@ -709,35 +734,237 @@ func TestTransactionHashDeployAccountTestnet(t *testing.T) {
 // Parameters:
 // - t: reference to the testing.T object
 // Returns:
-//  none
+//
+//	none
 func TestTransactionHashDeclare(t *testing.T) {
-	// https://goerli.voyager.online/tx/0x4e0519272438a3ae0d0fca776136e2bb6fcd5d3b2af47e53575c5874ccfce92
 	if testEnv != "testnet" {
-		t.Skip("Skipping test as it requires a testnet environment")
+		t.Skip("Skipping test as it requires a integration environment")
```

Comment:
We don't need to query the external environment to add tests for the hash functions.

Feel free to steal Juno's examples / test data for these.

Reply:
Currently the NewClient() function calls the external environment to get the Chain ID. Maybe we could use the mockProvider instead.

Reply:
I've updated the code to call the mockProvider instead of requiring integration.